MediaBrowser.Controller/Lyrics/LyricLine.cs

```diff
@@ -0,0 +1,28 @@
+namespace MediaBrowser.Controller.Lyrics;
+
+/// <summary>
+/// Lyric model.
+/// </summary>
+public class LyricLine
+{
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LyricLine"/> class.
+    /// </summary>
+    /// <param name="text">The lyric text.</param>
+    /// <param name="start">The lyric start time in ticks.</param>
+    public LyricLine(string text, long? start = null)
```

Comment:
Parameter order is not the same as the property definition order. Triggers my OCD :p